README.md

```diff
@@ -75,29 +75,29 @@ all the resources needed.
 
```

Comment:
Pre-requisities should this also be relative?

Reply:
We should add it for now :+1: